cypress/cypress/support/step_definitions/commons.ts

```diff
@@ -0,0 +1,338 @@
+import { Then, Given, When, DataTable, Step, BeforeStep } from "@badeball/cypress-cucumber-preprocessor";
+
+BeforeStep(() => { cy.wait(10); });
+
+Given('I visit {string}', (url: string) => {
+  cy.visit(url).then(() => {
+    cy.window().then((win) => {
+      return new Cypress.Promise((resolve) => {
+        if (win.document.readyState === 'complete') {
+          resolve();
+        } else {
+          win.addEventListener('load', resolve);
+        }
+      });
+    });
+  });
+});
+
+Then('I can read {string}', (title: string) => {
+  cy.contains(title).should('be.visible');
+});
+
+Then('I cannot see {string}', (button: string) => {
+  cy.contains(button).should('not.exist');
+});
+
+Then('I wait for the text {string} to appear', (text: string) => {
+  cy.contains(text).should('be.visible');
+});
+
+/* Button Step */
+
+When('I click on the {string} button', (name: string) => {
+  buttonClick(name, ['input', 'button', 'cds-button', 'cds-side-nav-link']);
+});
+
+When('I submit', () => {
+  buttonClick('Create client', ['input', 'button', 'cds-button', 'cds-side-nav-link'],'div.form-footer-group-buttons');
+});
```

Comment:
This would only work for the staff form...
Maybe we should select the button by this attribute `data-test="wizard-submit-button"` since all the three forms use this same value.
Or maybe we should remove this and just use `'I click on the {string} button'`.

Reply:
I would suggest waiting for this PR to be marked as ready before doing reviews as it is not close to being a final version yet